src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/ExpandedMenuToggle.tsx

```diff
@@ -44,52 +48,49 @@ const text = ({ showExpandedMenu }: { showExpandedMenu: boolean }) => css`
         display: inline-block;
         height: 8px;
         margin-left: 6px;
-        transform: ${showExpandedMenu
-            ? 'translateY(1px) rotate(-135deg)'
-            : 'translateY(-3px) rotate(45deg)'};
+
+        /*
+            IMPORTANT NOTE:
+            we need to specify the adjacent path to the a (current) tag
+            to apply styles to the nested tabs due to the face we use ~
+            to support NoJS
+        */
+        transform: translateY(-3px) rotate(45deg);
+        ${`#${CHECKBOX_ID}:checked ~ & {
+            transform: translateY(1px) rotate(-135deg);
+        }`}
```

Comment:
Why doesn't this work, why the double string interpolation?

```suggestion
        ${CHECKBOX_ID}:checked ~ & {
            transform: translateY(1px) rotate(-135deg);
        }
```